src/outputs/alsa.c

```diff
@@ -143,6 +143,8 @@ static int alsa_latency_history_size;
 static struct media_quality alsa_fallback_quality = { 44100, 16, 2, 0 };
 static struct media_quality alsa_last_quality;
 
+static int alsa_cfg_secn = 0;
```

Comment:
Drop the global and just call cfg_size() directly in alsa_session_make(). Also, globals are always initialized to zero automatically, so another time you can do without "= 0".